openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -100,10 +100,17 @@ instances:
   ## @param ssl_private_key - string - optional
   ## Needed if the certificate linked in ssl_cert does not include the private key.
   ## Note: The private key to your local certificate must be unencrypted.
-  # 
+  #
   #  ssl_private_key: "<KEY_PATH>"
 
-  ## @param ssl_ca_cert - string - optional 
+  ## @param ssl_ca_cert - string - optional
   ## The path to the trusted CA used for generating custom certificates.
   #
   #  ssl_ca_cert: "<CA_CERT_PATH>"
+
+  ## @param extra_headers - list of key:value elements - optional
+  ## A list of additional headers to send in queries to the openmetrics endpoint.
```

Comment:
Probably obvious to most, but better safe than escalate-y, should we say they are "HTTP headers"?

Reply:
👍 fixed